pontoon/base/utils.py

```diff
@@ -682,12 +681,14 @@ def glob_to_regex(glob):
     """
     pattern = PatternParser().parse(glob)
     regex = r""
-
     for part in pattern:
-        if isinstance(part, Starstar):
-            regex += r"(.*)"
-        elif isinstance(part, Star):
-            regex += r"([^/]*)"
+        if isinstance(part, Star):
```

Comment:
@Pike Hey, can you look at this part again? It looks very similar to the previous solution and I wasn't convinced if that's what you've meant (because it's a little bit late :hourglass::night_with_stars:).
Thanks in advance!